pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -534,7 +534,7 @@ def get_c_interface():
         #   lib.<system>-<architecture>-<python-id>"
         # To avoid mismatching the folder name, we search for the shared object instead.
         # TODO: locate where the naming convention of the folder is decided and replicate it here.
-        editable_mode_path = package_root.parent.parent / "build"
+        editable_mode_path = package_root.parent.parent / "build_lightning_kokkos"
```

Comment:
Shall we add a unit test for this editable_mode path?

Reply:
Second to it.

Reply:
I'll add one!